models/exceptions.py

```diff
@@ -49,4 +51,18 @@ def __init__(self,
 
     def __str__(self):
         """str definition"""
-        return (f'{self.message} {str(self.typeSetTo)}')
\ No newline at end of file
+        return (f'{self.message} {str(self.typeSetTo)}')
+
+
+class ApiReturnNoneResults(Exception):
+    """Exception raised when an API call return None events list"""
```

Comment:
reallllly predantic, but this should say "when API call returns a 200 response, but the response is None or Empty"